integration_tests/.scripts/integration_test.sh

```diff
@@ -10,7 +10,7 @@ do
   esac
 done
 
-declare -a SUPPORTED_DATABASES=("bigquery" "databricks" "postgres" "redshift" "snowflake")
+declare -a SUPPORTED_DATABASES=("bigquery" "postgres" "redshift" "snowflake")
```

Comment:
I think we can keep databricks in here as we never really call it was 'all' anyway.